docs/endpoints/circuits.md

```diff
@@ -0,0 +1,209 @@
+[← Documentation Home](/docs/README.md)
+# Circuits
+
+Gets list of circuits in alphabetical order by circuitId
+
+**URL** : `/ergast/f1/circuits/`
+
+[Available Query Parameters](./README.md#query-parameters)
+
+---
+
+## Route Parameters:
+
+### Season
+
+**Filters only circuits which hosted a race in a given seasons. Year numbers are valid as is `current` to get the current season list of circuits**
+
+`/{season}/` -> ex: `/ergast/f1/2024/circuits/`
+
+**Note**: To utilize the `season` parameter it needs to be the first route after `/ergast/f1/`
+
+---
+
+### Round
+
+**Filters only for the circuit that hosted the race in the specified round of the specific season. Round numbers 1 -> n races are valid as well as `last` and `next`**
+
+`/{round}/` -> ex: `/ergast/f1/2024/1/circuits/`
+
+**Note**: To utilize the `round` parameter it needs to be used with the `season` filter and be the first route after `/ergast/f1/{season}`
+
+---
+
+### circuits
+
+**Filters for only the circuit that matches the specified circuitId**
+
+`/circuits/{circuitId}/` -> ex: `/ergast/f1/2024/circuits/albert_park/circuits/`
+
+---
+
+### constructors
+
+**Filters for only circuits that the specified constructor has participated in a race at**
+
+`/constructors/{constructorId}/` -> ex: `/ergast/f1/constructors/williams/circuits/`
+
+---
+
+### drivers
+
+**Filters for only circuits that the specified driver has participated in a race at**
+
+`/drivers/{driverId}/` -> ex: `/ergast/f1/drivers/hamilton/circuits/`
+
+
+---
+
+### fastest
+
+**Filters for only circuits that have had a race with a lap that was the ranked in the specified position**
+
+`/fastest/{lapRank}/` -> ex: `/ergast/f1/fastest/24/circuits/`
+
+
+---
+
+### grid
+
+**Filters for only circuits that have had a race with a specific grid position**
+
+`/grid/{gridPosition}/` -> ex: `/ergast/f1/grid/29/circuits/`
+
+---
+
+### results
+
+**Filters for only circuits that have had a race where a specific finishing position was valid**
+
+`/results/{finishPosition}/` -> ex: `/ergast/f1/results/1/circuits/`
+
+---
+
+### status
+
+**Filters for only circuits that have had a race where a driver finished with a specific statusId**
+
+`/status/{statusId}/` -> ex: `/ergast/f1/status/2/drivers/`
+
+---
+
+## Success Response
+
+**Code** : `200 OK`
+
+**Response Fields** :
+
+[Common Response Fields](./README.md#common-response-fields)
+
+`MRData.CircuitTable` : The object containing the list of the all drivers.
+
+`MRData.CircuitTable.Circuits` : The list of all drivers returned.
+
+`MRData.CircuitTable.Circuits[i]` : A given driver object
+
+---
+
+## Circuits Object Fields:
+
+|Field|Required|
+|---|:---:|
+|circuitId| ✅ |
+|url|✅|
+|circuitName|✅|
+|Location|✅|
```

Comment:
```suggestion
|circuitId| 🟡 |
|url|🟡|
|circuitName|✅|
|Location|🟡|
```
New option of yellow, may be best. We 99% of time provide this, but from a database perspective there could be some times where the fields are missing.

Reply:
Good point. I like the yellow instead